CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* base: Pmac module version update. by @guirodrigueslima
```

Comment:
This should be under a "Bug fixes" or "New features" subheader, and is missing a link to the PR.